custom_components/homewhiz/__init__.py

```diff
@@ -1,111 +1,114 @@
-import asyncio
 import logging
 
 from bleak import BleakClient
 from bleak.backends.device import BLEDevice
+from bleak_retry_connector import establish_connection
```

Comment:
Maybe we should add this as a requirement to the manifest?

Reply:
I'm not sure about that, because `bleak_retry_connector` is a leary shipped with homeassiatant, but better safe than sorry

Reply:
I agree - especially since the same applies to bleak which is also in the manifest! Thanks 👍